ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -274,6 +274,13 @@ def astype(self, dtype):
     def logical_xor(self, y, out=None, name=None):
         return paddle_frontend.logical_xor(self, y, out=out)
 
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("float16", "float32", "float64", "int32", "int64")},
+        "paddle",
+    )
+    def isnan(self, name=None):
+        return ivy.isnan(self._ivy_array)

```

Comment:
Could you please change this to `paddle_frontent.isnan()`

Reply:
sure

Reply:
@aparajith21 done